CI/travis/before_install_linux

```diff
@@ -21,7 +21,7 @@ handle_default() {
 	sudo apt-get -qq update
 	sudo apt-get install -y cmake graphviz libaio-dev libavahi-client-dev \
 		libavahi-common-dev libusb-1.0-0-dev libxml2-dev rpm tar \
-		bzip2 gzip flex bison git python-pip
+		bzip2 gzip flex bison git python-pip wget
```

Comment:
`wget` should already be installed via https://github.com/analogdevicesinc/libiio/blob/master/CI/travis/lib.sh#L317